cartoframes/context.py

```diff
@@ -546,6 +573,28 @@ def query(self, query, table_name=None, decode_geom=False, is_select=None):
               By default `is_select=None` that means that the method will return a dataframe if
               the `query` starts with a `select` clause, otherwise it will just execute the query
               and return `None`
+            query_columns (list, optional): A list of `cartoframes.column.Column` to infer data types. You should
+              create `Column` instances this way: Column('column_name', pgtype='the_database_type') where
+              `the_database_type` is mapped using this dictionary:
+
+              .. code:: python
+
+                {'bigint': 'float64',
+                 'boolean': 'bool',
+                 'date': 'datetime64[ns]',
+                 'double precision': 'float64',
+                 'geometry': 'object',
+                 'int': 'int64',
+                 'integer': 'float64',
+                 'number': 'float64',
+                 'numeric': 'float64',
+                 'real': 'float64',
+                 'smallint': 'float64',
+                 'string': 'object',
+                 'timestamp': 'datetime64[ns]',
+                 'timestamp with time zone': 'datetime64[ns]',
```

Comment:
`timestamptz` is also valid

Reply:
Looking good :) To me it's important to have `timestamptz` since you can cast values in pg using the casting operator (`::`).
<img width="732" alt="Screen Shot 2019-04-19 at 11 20 50 AM" src="https://user-images.githubusercontent.com/1041056/56437694-3deb7100-6295-11e9-8e83-068b9fb6283d.png">